Podfile.lock

```diff
@@ -1,16 +1,16 @@
 PODS:
-  - AWSAuthCore (2.24.0):
-    - AWSCore (= 2.24.0)
-  - AWSCognitoIdentityProvider (2.24.0):
-    - AWSCognitoIdentityProviderASF (= 2.24.0)
-    - AWSCore (= 2.24.0)
-  - AWSCognitoIdentityProviderASF (2.24.0)
-  - AWSCore (2.24.0)
-  - AWSMobileClient (2.24.0):
-    - AWSAuthCore (= 2.24.0)
-    - AWSCognitoIdentityProvider (= 2.24.0)
-    - AWSCognitoIdentityProviderASF (= 2.24.0)
-    - AWSCore (= 2.24.0)
+  - AWSAuthCore (2.24.4):
+    - AWSCore (= 2.24.4)
+  - AWSCognitoIdentityProvider (2.24.4):
+    - AWSCognitoIdentityProviderASF (= 2.24.4)
+    - AWSCore (= 2.24.4)
+  - AWSCognitoIdentityProviderASF (2.24.4)
+  - AWSCore (2.24.4)
+  - AWSMobileClient (2.24.4):
+    - AWSAuthCore (= 2.24.4)
+    - AWSCognitoIdentityProvider (= 2.24.4)
+    - AWSCognitoIdentityProviderASF (= 2.24.4)
+    - AWSCore (= 2.24.4)
```

Comment:
do we need to update the rest of the lock files?

Reply:
Added